src/particle.jl

```diff
@@ -73,7 +112,19 @@ is_neutrino(::Type{PNuTau}) = true
 is_neutrino(::Type{PNuTauBar}) = true
 
 
+"""
+    Particle{T,PType<:ParticleType}
+
+Struct containing information of a single particle inside the detector
```

Comment:
Struct containing information of a single particle. 
(doesn't have to be inside the detector)